project-cli/src/templates/styles.mustache

```diff
@@ -1,3 +1,3 @@
-.{{fileName}} {
+.{{styleName}} {
```

Comment:
This works fine for class/functional components, but it is empty when we generate `page`.